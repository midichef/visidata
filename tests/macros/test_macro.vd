sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	open input in VisiData
			exec-1
			save-sheet	tests/macros/golden/test_macro.tsv
			quit-all
