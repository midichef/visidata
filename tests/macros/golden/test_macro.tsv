OrderDate	Region	Rep	Item	Units	Unit_Cost	Total
2016-01-06	East	Jones	Pencil	95	1.99	189.05
2016-01-23	Central	Kivell	Binder	50	19.99	999.50
2016-02-09	Central	Jardine	Pencil	36	4.99	179.64
2016-02-26	Central	Gill	Pen	27	19.99	539.73
2016-03-15	West	Sorvino	Pencil	56	2.99	167.44
2016-04-01	East	Jones	Binder	60	4.99	299.40
